src/neo/SmartContract/ApplicationEngine.cs

```diff
@@ -51,6 +52,20 @@ private bool AddGas(long gas)
             return testMode || GasConsumed <= gas_amount;
         }
 
+        public void StorageReused(int reusedBytes)
+        {
+            var discountPerByte = NativeContract.Policy.GetDiscountPerByteReused(Snapshot);
```

Comment:
https://github.com/neo-project/neo/blob/8eef1ac7b3936ab34e2a2a707c83a48182ca774a/src/neo/SmartContract/InteropService.cs#L21

Use `GasPerByte`?

Reply:
Do you mean using this property directly? 
Shouldn't this GasPerByte be on the network policy?

Reply:
Will we change it?

Reply:
Hmm, but if the fees are given to the consensus nodes, shouldn't they be responsible for updating this fee?
Maybe this is a little out of the scope of this PR. My concern is that we will *never* be allowed to change the fee. 


About the PR itself, I can move to variables, I think that we can pay back 1:1 when users release storage, but when they reuse storage too? What if we can pay back a little more when the storage is being released since the user is also charged by the opcode?
How "attractive" should be this storage release?

I added the values to the policy because I think it will be hard to come up with value easily.  I don't really know the best ratio for this payback.

Reply:
The fee is pay to the neo holders, not the consensus nodes.

Reply:
Yes, sorry. Reading NetworkPolicy everywhere that I thought this was a network fee value. 

About the difference between reuse and release, do you still prefer one single variable (GasPerByte) or should I add one or two more variables?

Reply:
I don't know. The core developers should discuss it again.